views/home.dt

```diff
@@ -90,6 +90,13 @@ block body
 				li.selected= i+1
 			- else
 				li: a(href="?sort=#{sort_mode}&category=#{category}&skip=#{i*info.limit}&limit=#{info.limit}")= i+1
+	ul.pageNav.perPage
+		- static immutable limit_enum = [10, 20, 50, 100];
+		- foreach (limit; limit_enum)
+			- if (info.limit == limit)
+				li.selected= limit
+			- else
+				li: a(href="?sort=#{sort_mode}&category=#{category}&skip=#{info.skip}&limit=#{limit}")= limit
```

Comment:
Ehm I guess at some point one should add a function that avoid these error-prone duplications.